internal/server/logbuffer/merge.go

```diff
@@ -0,0 +1,153 @@
+package logbuffer
+
+import (
+	"io"
+	"time"
+)
+
+// LogMerge can combine multiple log streams into one stream. It presumes
+// each stream emits entries in time order, and then weaves the entries
+// together to create a total time ordered stream.
+type LogMerge struct {
```

Comment:
Given the name of the package is `logbuffer` and has "log" in it, I think this type should probably be named `Merger` or something. This is I think a tao of idiomatic Go or Effective Go or w/e.

Reply:
`type ThatWhichMerges struct .... ` 🤣 

Totes, I'll make this Merger, that's a better name.